bitcoin/src/blockdata/transaction.rs

```diff
@@ -1030,6 +1030,12 @@ impl Decodable for Version {
     }
 }
 
+impl fmt::Display for Version {
+    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
+        write!(f, "Version({})", self.0)
```

Comment:
In b495a2b7ce5dd13447a67e5b80cbfcc26a4df69d:

I think we should just pass through to the underlying u32, like `fmt::Display::fmt(&self.0, f)`. This would preserve all formatting options without making us think explicitly about it. I also don't think that `Display` should add any extra stuff like `Version()`.

Reply:
Got it